EF/DeliveryService/DeliveryService/Models/Basket.cs

```diff
@@ -0,0 +1,29 @@
+﻿using DeliveryService.Models.BaseModel;
+using System.Collections.Generic;
+
+
+namespace DeliveryService.Models
+{
+    public class Basket : Model
+    {
+        public Client Client { get; set; }
```

Comment:
add ClientId

Reply:
In this task has not used entity framework yet , can I  leave only a link to the object ?

Reply:
yeah